services/cognitiveservices/v1.0/customsearch/version.go

```diff
@@ -21,10 +19,10 @@ import "github.com/Azure/azure-sdk-for-go/version"
 
 // UserAgent returns the UserAgent string to use when sending http.Requests.
 func UserAgent() string {
-	return "Azure-SDK-For-Go/" + version.Number + " customsearch/1.0"
+	return "Azure-SDK-For-Go/v14.0.0 services"
 }
 
 // Version returns the semantic version (see http://semver.org) of the client.
 func Version() string {
-	return version.Number
+	return "v14.0.0"
```

Comment:
~~This ought to be "v14.1.0".~~

edit: This ought to be referencing the central version definition.

Reply:
Guessing the --use-onever flag was omitted when the generator was invoked.